### ISSUE TITLE
KEditor breaks background images on save.

### ISSUE THREAD

**Issue by stephen-fueldesign** [Kademi/kademi-dev#5866]
Quite often when I'm editing a background image on either a component or section the background images become corrupted. Especially over time.

It's because of the inline css.

**Normally the pure CSS is**:
`background-image:url("/asset/c18b42cb-2f76-4ed0-aef1-f89bae7cb918");`

**But because it's inline CSS, keditor does the following**:
`style="background-image:url(&quote;/asset/xxx&quote;)"`

**But then it goes crazy after you save the page again**:
`<div class="keditor-component-text-content" style="min-height: 288px; background-image: url(" assets="" 9492c418-9e96-4793-aa81-43269e6e69a2");"="">`

**Proposed Solution**:
`style="background-image: url('/assets/c18b42cb-2f76-4ed0-aef1-f89bae7cb918');"`

**Comment by bradmac** [comment]
@stephen-fueldesign are you entering the background through component settings?

**Comment by stephen-fueldesign** [comment]
Yes, using keditor as expected; changing background image works first time round. Then when you save it degrades over time.

**Comment by anhnt** [comment]
I cant reproduce this bug. 
FYI browser is always using double quotes `"` when we set inline css via js
I think there should be one-off case when there was some other js error that caused the problem
